apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -90,6 +91,11 @@ async function getTemplateQuestions(questions: QuestionsPageData[]) {
 
 router.get(
   '/',
+  createAuthzMiddleware({
+    oneOfPermissions: ['has_course_permission_preview'],
+    errorMessage: 'Requires "Previewer" permissions',
+    unauthorizedUsers: 'passthrough',
+  }),
```

Comment:
This is an interesting case, this page will incorrectly suggest that "student data view" is sufficient for this page because of this middleware on the entire route prefix here:

https://github.com/PrairieLearn/PrairieLearn/blob/f56a37578d3c6d2dedf1442cb02fc1e0c060bcaf/apps/prairielearn/src/server.ts#L650-L652

The one in this will still correctly enforce the more specific permission requirements, so I'm not too worried about this, I just happened to notice this during testing.